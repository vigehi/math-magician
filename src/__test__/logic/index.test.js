import React from 'react';
import { render } from '@testing-library/react';
import Mainapp from '../../mainapp';
import Header from '../../header';
import Home from '../../home';
import Quotes from '../../quote';

test('snapshot of mainapp', () => {
  const renderedComponat = render(<Mainapp />);
  expect(renderedComponat).toMatchSnapshot();
});

test('snapshot of header', () => {
  const renderedComponat = render(<Header />);
  expect(renderedComponat).toMatchSnapshot();
});

test('snapshot of mainapp', () => {
  const renderedComponat = render(<Home />);
  expect(renderedComponat).toMatchSnapshot();
});

test('snapshot of mainapp', () => {
  const renderedComponat = render(<Quotes />);
  expect(renderedComponat).toMatchSnapshot();
});
