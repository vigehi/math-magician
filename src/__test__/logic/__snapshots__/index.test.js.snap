// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of header 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header>
        <h1>
          Edith Magic
        </h1>
        <ul>
          <a
            href="/"
          >
            Home
          </a>
          <a
            href="/Calculator"
          >
            Calculator
          </a>
          <a
            href="/Quotes"
          >
            Quotes
          </a>
        </ul>
      </header>
    </div>
  </body>,
  "container": <div>
    <header>
      <h1>
        Edith Magic
      </h1>
      <ul>
        <a
          href="/"
        >
          Home
        </a>
        <a
          href="/Calculator"
        >
          Calculator
        </a>
        <a
          href="/Quotes"
        >
          Quotes
        </a>
      </ul>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`snapshot of mainapp 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <header>
          <h1>
            Edith Magic
          </h1>
          <ul>
            <a
              href="/"
            >
              Home
            </a>
            <a
              href="/Calculator"
            >
              Calculator
            </a>
            <a
              href="/Quotes"
            >
              Quotes
            </a>
          </ul>
        </header>
        <h1>
          Magic...... Lets go
        </h1>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla dignissim, dolor id sagittis tincidunt, mauris mauris consequat leo, et commodo purus est vitae ex. Duis nisi lacus, facilisis in varius ac, pulvinar nec libero. Pellentesque feugiat elementum bibendum. Ut convallis libero sit amet tortor egestas, nec laoreet eros ullamcorper. Nullam laoreet semper rutrum. Mauris leo enim, vestibulum ut lacus sed, blandit imperdiet nibh. Donec eget velit nec ipsum auctor egestas. Nullam eget lacinia sapien. Vivamus faucibus ut nibh vel tristique. Aliquam rutrum eros a augue malesuada, a egestas tortor rhoncus Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla dignissim, dolor id sagittis tincidunt, mauris mauris consequat leo, et commodo purus est vitae ex. Duis nisi lacus, facilisis in varius ac, pulvinar nec libero. Pellentesque feugiat elementum bibendum. Ut convallis libero sit amet tortor egestas, nec laoreet eros ullamcorper. Nullam laoreet semper rutrum. Mauris leo enim, vestibulum ut lacus sed, blandit imperdiet nibh. Donec eget velit nec ipsum auctor egestas. Nullam eget lacinia sapien. Vivamus faucibus ut nibh vel tristique. Aliquam rutrum eros a augue malesuada, a egestas tortor rhoncus
        </p>
        <img
          alt="img"
          src="mao.png"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <header>
        <h1>
          Edith Magic
        </h1>
        <ul>
          <a
            href="/"
          >
            Home
          </a>
          <a
            href="/Calculator"
          >
            Calculator
          </a>
          <a
            href="/Quotes"
          >
            Quotes
          </a>
        </ul>
      </header>
      <h1>
        Magic...... Lets go
      </h1>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla dignissim, dolor id sagittis tincidunt, mauris mauris consequat leo, et commodo purus est vitae ex. Duis nisi lacus, facilisis in varius ac, pulvinar nec libero. Pellentesque feugiat elementum bibendum. Ut convallis libero sit amet tortor egestas, nec laoreet eros ullamcorper. Nullam laoreet semper rutrum. Mauris leo enim, vestibulum ut lacus sed, blandit imperdiet nibh. Donec eget velit nec ipsum auctor egestas. Nullam eget lacinia sapien. Vivamus faucibus ut nibh vel tristique. Aliquam rutrum eros a augue malesuada, a egestas tortor rhoncus Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla dignissim, dolor id sagittis tincidunt, mauris mauris consequat leo, et commodo purus est vitae ex. Duis nisi lacus, facilisis in varius ac, pulvinar nec libero. Pellentesque feugiat elementum bibendum. Ut convallis libero sit amet tortor egestas, nec laoreet eros ullamcorper. Nullam laoreet semper rutrum. Mauris leo enim, vestibulum ut lacus sed, blandit imperdiet nibh. Donec eget velit nec ipsum auctor egestas. Nullam eget lacinia sapien. Vivamus faucibus ut nibh vel tristique. Aliquam rutrum eros a augue malesuada, a egestas tortor rhoncus
      </p>
      <img
        alt="img"
        src="mao.png"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`snapshot of mainapp 2`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <header>
          <h1>
            Edith Magic
          </h1>
          <ul>
            <a
              href="/"
            >
              Home
            </a>
            <a
              href="/Calculator"
            >
              Calculator
            </a>
            <a
              href="/Quotes"
            >
              Quotes
            </a>
          </ul>
        </header>
        <h1>
          Magic...... Lets go
        </h1>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla dignissim, dolor id sagittis tincidunt, mauris mauris consequat leo, et commodo purus est vitae ex. Duis nisi lacus, facilisis in varius ac, pulvinar nec libero. Pellentesque feugiat elementum bibendum. Ut convallis libero sit amet tortor egestas, nec laoreet eros ullamcorper. Nullam laoreet semper rutrum. Mauris leo enim, vestibulum ut lacus sed, blandit imperdiet nibh. Donec eget velit nec ipsum auctor egestas. Nullam eget lacinia sapien. Vivamus faucibus ut nibh vel tristique. Aliquam rutrum eros a augue malesuada, a egestas tortor rhoncus Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla dignissim, dolor id sagittis tincidunt, mauris mauris consequat leo, et commodo purus est vitae ex. Duis nisi lacus, facilisis in varius ac, pulvinar nec libero. Pellentesque feugiat elementum bibendum. Ut convallis libero sit amet tortor egestas, nec laoreet eros ullamcorper. Nullam laoreet semper rutrum. Mauris leo enim, vestibulum ut lacus sed, blandit imperdiet nibh. Donec eget velit nec ipsum auctor egestas. Nullam eget lacinia sapien. Vivamus faucibus ut nibh vel tristique. Aliquam rutrum eros a augue malesuada, a egestas tortor rhoncus
        </p>
        <img
          alt="img"
          src="mao.png"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <header>
        <h1>
          Edith Magic
        </h1>
        <ul>
          <a
            href="/"
          >
            Home
          </a>
          <a
            href="/Calculator"
          >
            Calculator
          </a>
          <a
            href="/Quotes"
          >
            Quotes
          </a>
        </ul>
      </header>
      <h1>
        Magic...... Lets go
      </h1>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla dignissim, dolor id sagittis tincidunt, mauris mauris consequat leo, et commodo purus est vitae ex. Duis nisi lacus, facilisis in varius ac, pulvinar nec libero. Pellentesque feugiat elementum bibendum. Ut convallis libero sit amet tortor egestas, nec laoreet eros ullamcorper. Nullam laoreet semper rutrum. Mauris leo enim, vestibulum ut lacus sed, blandit imperdiet nibh. Donec eget velit nec ipsum auctor egestas. Nullam eget lacinia sapien. Vivamus faucibus ut nibh vel tristique. Aliquam rutrum eros a augue malesuada, a egestas tortor rhoncus Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla dignissim, dolor id sagittis tincidunt, mauris mauris consequat leo, et commodo purus est vitae ex. Duis nisi lacus, facilisis in varius ac, pulvinar nec libero. Pellentesque feugiat elementum bibendum. Ut convallis libero sit amet tortor egestas, nec laoreet eros ullamcorper. Nullam laoreet semper rutrum. Mauris leo enim, vestibulum ut lacus sed, blandit imperdiet nibh. Donec eget velit nec ipsum auctor egestas. Nullam eget lacinia sapien. Vivamus faucibus ut nibh vel tristique. Aliquam rutrum eros a augue malesuada, a egestas tortor rhoncus
      </p>
      <img
        alt="img"
        src="mao.png"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`snapshot of mainapp 3`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <header>
          <h1>
            Edith Magic
          </h1>
          <ul>
            <a
              href="/"
            >
              Home
            </a>
            <a
              href="/Calculator"
            >
              Calculator
            </a>
            <a
              href="/Quotes"
            >
              Quotes
            </a>
          </ul>
        </header>
        <img
          alt="img"
          class="img"
          src="in.png"
        />
        <h4>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla dignissim, dolor id sagittis tincidunt, mauris mauris consequat leo, et commodo purus est vitae ex. Duis nisi lacus, facilisis in varius ac, pulvinar nec libero. Pellentesque feugiat elementum bibendum. Ut convallis libero sit amet tortor egestas, nec laoreet eros ullamcorper. Nullam laoreet semper rutrum. Mauris leo enim, vestibulum ut lacus sed, blandit imperdiet nibh. Donec eget velit
        </h4>
        <img
          alt="img"
          src="mao.png"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <header>
        <h1>
          Edith Magic
        </h1>
        <ul>
          <a
            href="/"
          >
            Home
          </a>
          <a
            href="/Calculator"
          >
            Calculator
          </a>
          <a
            href="/Quotes"
          >
            Quotes
          </a>
        </ul>
      </header>
      <img
        alt="img"
        class="img"
        src="in.png"
      />
      <h4>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla dignissim, dolor id sagittis tincidunt, mauris mauris consequat leo, et commodo purus est vitae ex. Duis nisi lacus, facilisis in varius ac, pulvinar nec libero. Pellentesque feugiat elementum bibendum. Ut convallis libero sit amet tortor egestas, nec laoreet eros ullamcorper. Nullam laoreet semper rutrum. Mauris leo enim, vestibulum ut lacus sed, blandit imperdiet nibh. Donec eget velit
      </h4>
      <img
        alt="img"
        src="mao.png"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
